bears/go/GoVetBear.py

```diff
@@ -25,5 +28,96 @@ class GoVetBear:
     CAN_DETECT = {'Unused Code', 'Smell', 'Unreachable Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return 'vet', filename
+    def create_arguments(filename, file, config_file,
+                         assembly_go_mismatch: bool=False,
+                         useless_assignment: bool=False,
```

Comment:
We should check for useless assignments by default :)
Also -> `useless_assignments` (plural s)

Reply:
Maybe add `check_` prefix or `allow_`/`disallow_` prefix to many settings to clarify when something is allowed or disallowed.

Reply:
@Makman2 https://github.com/coala/coala-bears/pull/2257#issuecomment-360585568
Every setting is true by default. ;)

Reply:
Yeah, I think those prefixes should be added to all of them.